Formula/raylib.rb

```diff
@@ -0,0 +1,44 @@
+class Raylib < Formula
+  desc "Simple and easy-to-use library to learn videogames programming"
+  homepage "http://www.raylib.com"
+  url "https://github.com/raysan5/raylib/archive/1.9.1-dev.tar.gz"
```

Comment:
needs to be a non-dev release

Reply:
> Formula depends on build system changes that aren't in master, therefore
stable points to the oldest development tag that has the changes
and --HEAD points to the develop branch.

Reply:
`-dev` implies not a stable release

Reply:
It's an intermittent release I asked for, because the author want publish a new one till mid 2018 and the stable version a month ago lacks changes done to the build system. I will upgrade this to the proper 2.0 release when that one happens.

Reply:
OK. We can waive the usual requirement that stable-only releases be used if upstream released it specifically for this.